### ISSUE TITLE
MaskGit forward() - "ids" is fed to transformer rather than "x"

### ISSUE THREAD

**Issue by lonzi** [lucidrains/muse-maskgit-pytorch#11]
Hi, 
Thanks a lot for the implementation.

While running the base MaskGIT training according to the usage example:
```
loss = base_maskgit(
    images,
    texts = texts
)

```

I noticed that the training cross entropy loss goes down to very small values pretty fast. I suspect this is related to the following line of code:
![image](https://user-images.githubusercontent.com/30550331/212644824-a5a871de-26ec-4846-88b9-067407616fbc.png)

https://github.com/lucidrains/muse-maskgit-pytorch/blob/main/muse_maskgit_pytorch/muse_maskgit_pytorch.py#L513

where _ids_ is passed to the transformer forward function rather than the masked _x_. 

Am I missing something? thanks :-)

**Comment by lucidrains** [comment]
@lonzi i'm just testing to see if people are paying attention

just kidding :laughing: my bad! thank you for catching this :pray:

**Comment by lucidrains** [comment]
@lonzi should be fixed in 0.0.19!

**Comment by lonzi** [comment]
great. thanks!
